### PR TITLE
5. View an individual review + add loading screen

### PR DESCRIPTION
**Pull request by impemberton**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,22 +1,34 @@
...
 
 const App = () => {
+  const navigate = useNavigate();
+  const [loading, setLoading] = useState(true);
+  useEffect(() => {
+    setLoading(true);
+    setTimeout(() => {
+      setLoading(false);
+    }, 800);
+  }, [navigate]);
```
Comment (pkaodev) [inline src/App.js]: I don't think navigate is doing anything here in the dependency array.  It is a function, not a state.

Comment (pkaodev) [reply src/App.js]: Is navigate being used at all?  If not remove it

File: src/Pages/Review/Review.jsx [hunk 2]
```diff
@@ -1,10 +1,36 @@
...
-import './Review.css'
+import React, { useEffect, useState } from "react";
+import { useLocation } from "react-router-dom";
+import { getReview } from "../../utils";
+import "./Review.css";
 
 const Review = () => {
+  const [currReview, setCurrReview] = useState({});
+  const location = useLocation();
+  console.log(location.pathname.split("/").reverse()[0])
```
Comment (pkaodev) [inline src/Pages/Review/Review.jsx]: You could do this more cleanly with the useParams hook

File: src/App.js (hunk 1 above)
Comment (pkaodev) [inline src/App.js]: I'm not sure adding a 800ms forced loading message is the way to go.  Loading should be happening when we are waiting for responses from the api.   Let's remove it for now

File: src/Pages/Review/Review.jsx [hunk 3]
```diff
@@ -1,10 +1,36 @@
...
+    getReview(location.pathname.split("/").reverse()[0]).then((data) =>
+      setCurrReview(data.review)
+    );
+  }, [location]);
+  console.log(currReview)
   return (
-    <div>Review</div>
-  )
-}
+    <div className="App__Review">
```
Comment (pkaodev) [inline src/Pages/Review/Review.jsx]: here we DO want a loading message which shows before getReview has returned information to us

**Comment by pkaodev** [comment]
Good start but we have a few things to change:
-remove the fake loading message
-add a loading state + message for your Review component
-(if you want to) use the useParams hook to get the review_id

Submit another PR with these changes.

**Comment by cjplatten** [comment]
These changes look good!

**Pull request merged**